contracts/OperatorFilter/DefaultOperatorFilterer.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
```

Comment:
any reason not to use the npm package https://www.npmjs.com/package/operator-filter-registry?

Reply:
This is based on your previous diff: https://github.com/magiceden-oss/erc721m/pull/70/commits/62e3af54853d93ef85c5a23d7cd5b863d0089f9e

I forgot the reason to have our own. The code seems simpler than the OS version.